app/queries/webhooks_query.rb

```diff
@@ -21,14 +18,18 @@ def call
 
     result.webhooks = webhooks
     result
+  rescue BaseService::FailedResult
+    result
```

Comment:
same question, what is being rescued here... I haven't seen this rescue in other query objects.